code/game/objects/items.dm

```diff
@@ -345,6 +345,29 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 
 			else if(S.can_be_inserted(src))
 				S.handle_item_insertion(src)
+		return 
+	if (istype(user,/mob/living/carbon/human))

```

Comment:
https://github.com/tgstation/tgstation/blob/4bc5e1d085de4f89b428515a8fdc6cf2823198f4/code/modules/mob/living/simple_animal/friendly/drone/_drone.dm#L64

Can you just stop being a tool.